cmd/glasskube/cmd/output_options.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type OutputFormat string
```

Comment:
`OutputFormat` should implement [`pflag.Value`](https://pkg.go.dev/github.com/spf13/pflag@v1.0.5#Value). To do this, you have to create the `String()`, `Set(string)` and `Type()` functions. `ListFormat` already has those functions you can just copy them. 

You can then use `flags.VarP(...)` instead of `flags.StringVarP(...)` with an explicit cast. 

Finally, please adapt the list command to also use `OutputOptions` and remove the `ListFormat` type, since it should be no longer used by then.

Again, let me know if you have any questions or require assistance!